lib/src/data/repositories/account_repository.dart

```diff
@@ -12,7 +12,7 @@ final accountRepositoryProvider = Provider<AccountRepository>(
 abstract class AccountRepository {
   Future<Account> get();
 
-  Future<Account> patch({required String name, required String email, required String password, required String avatar});
+  Future<Account> patch(Account account);
```

Comment:
I would prefer a named parameter here to be consistent with the other repositories.

Reply:
Sneaky gemarked als resolved zonder aan te passen of feedback te geven waarom je het niet wil aanpassen. 😏 @Pekket

Reply:
Made argument a named argument in 51190b7dc379ab4990c325243aa6370ac371428c.